apps/test/unit/templates/sectionProgress/SectionProgressToggleTest.js

```diff
@@ -23,6 +25,19 @@ describe('SectionProgressToggle', () => {
     expect(wrapper.find('#uitest-standards-toggle').exists()).to.be.true;
   });
 
+  it('sends toggle event when level is clicked', () => {
+    const wrapper = shallow(
+      <UnconnectedSectionProgressToggle {...DEFAULT_PROPS} />
+    );
+    const analyticsSpy = sinon.spy(analyticsReporter, 'sendEvent');
+
+    wrapper.instance().onChange();
```

Comment:
oof this feels like an anti pattern but it looks like it might be difficult to trigger any other way :(

Reply:
I agree.. I had a really hard time testing these considering the way the dashboard is set up. Better to leave it or just remove the test altogether?

Reply:
I'd leave it! It can just make it difficult to move these components into functional components in the future. But that's not a requirement so it's okay